stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpService.java

```diff
@@ -17,34 +17,61 @@
 */
 package org.ballerinalang.net.http;
 
+import org.ballerinalang.connector.api.Annotation;
+import org.ballerinalang.connector.api.BallerinaConnectorException;
+import org.ballerinalang.connector.api.Resource;
 import org.ballerinalang.connector.api.Service;
+import org.ballerinalang.connector.api.Struct;
+import org.ballerinalang.net.uri.DispatcherUtil;
 import org.ballerinalang.net.uri.URITemplate;
 import org.ballerinalang.net.uri.URITemplateException;
 import org.ballerinalang.net.uri.parser.Literal;
+import org.ballerinalang.util.exceptions.BallerinaException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.wso2.transport.http.netty.message.HTTPCarbonMessage;
 
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.ballerinalang.net.http.HttpConstants.HTTP_PACKAGE_PATH;
 
 /**
  * {@code HttpService} This is the http wrapper for the {@code Service} implementation.
  *
  * @since 0.94
  */
 public class HttpService {
+
+    private static final Logger log = LoggerFactory.getLogger(HttpService.class);
+
+    private static final String ENDPOINTS_FIELD = "endpoints"; // TODO: populate this in the service
+    private static final String LIFETIME_FIELD = "lifetime"; // TODO: populate this in the service
```

Comment:
Do we need the above unused constants?

Reply:
For the time being, no, since the endpoints and lifetime structs are not set in the config struct